README.md

```diff
@@ -100,9 +100,14 @@ val response: Response = httpPost {
             "email" to "john.doe@gmail.com" //  email=john.doe@gmail.com
         }
     }
+    // or
+    body {
+        form("login=user&email=john.doe@gmail.com")
```

Comment:
Have a feeling that it’s better to provide a form content value outside httpPost { } scope to provide a more valuable case.
How do u think @shtykh

Reply:
I don't see a lot of value in it. Can you make an example please?